lib/streamlit/server/server.py

```diff
@@ -344,11 +344,6 @@ def start(self, on_started: Callable[["Server"], Any]) -> None:
 
         self._ioloop.spawn_callback(self._loop_coroutine, on_started)
 
-    def get_debug(self) -> Dict[str, Dict[str, Any]]:
-        if self._session_data:
-            return {"report": self._session_data.get_debug()}
-        return {}
-
```

Comment:
Is this for a different PR?

Reply:
This was removed because the corresponding method for `session_data` was removed during its cleanup.